### ISSUE TITLE
[Chapter 16] How to visualize word embeddings in TensorBoard?

### ISSUE THREAD

**Issue by insuquot** [ageron/handson-ml2#172]
Hi, I'm new to Tensorflow and Keras--really enjoying your book!

How do you retrieve the 11,000 words (vocab_size + num_oov_buckets) from the `StaticVocabularyTable` in cell 54? At the end of the Sentiment Analysis section of chapter 16, you mention the possibility of using the TensorBoard() callback to watch the embeddings learned.

If I open TensorBoard Projector, I can see the word IDs, but not the words themselves. To do so, I tried to save the 10,000 vocabulary words to a TSV file, but I get an error message asking for 11,000 words. How can I retrieve all the words, especially the 1,000 OOV words?

**Comment by insuquot** [comment]
@ageron would love to get your thoughts on this

**Comment by IvanKuchin** [comment]
Hello @insuquot 

You may use Load feature of TensorBoard to upload as many words as you want to visualize and labels. Looks like you are familiar with Embedding Projector, but JFYI https://www.tensorflow.org/tutorials/text/word_embeddings
